test/unit/test-custom-element.js

```diff
@@ -1995,8 +1995,8 @@ describes.realWin('CustomElement', {amp: true}, (env) => {
       it('should NOT turn on when enters viewport but already laid out', () => {
         stubInA4A(false);
         const toggle = env.sandbox.spy(element, 'toggleLoading');
-        element.layoutCount_ = 1;
         element.viewportCallback(true);
+        element.layoutCount_ = 1;
```

Comment:
This should probably be its own unit test (one for each conditional).

The outer conditional check should check that both `toggleLoading` and `timer.delay` are not called.

Reply:
I'll add the test and the submit when its green.

Reply:
Sure. Added